### PR TITLE
Add tests, some refactoring

### PR DESCRIPTION
**Pull request by Lemeri02**

### CODE REVIEW

File: .github/workflows/python-package.yml [hunk 1]
```diff
@@ -42,22 +42,30 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: 'Create env file'
+    - name: 'Create server env file'
```
Comment (AkininD) [inline .github/workflows/python-package.yml:45]: Great!

File: .github/gener_sample.png
Comment (AkininD) [inline .github/gener_sample.png:1]: great job !

File: .github/conv_sample.png
Comment (AkininD) [inline .github/conv_sample.png:1]: Good!

File: .github/workflows/python-package.yml [hunk 2]
```diff
@@ -42,22 +42,30 @@ jobs:
...
     - name: Test server app with pytest
       working-directory: ./server
       run: |
         pytest
-#    - name: Test bot app with pytest
-#      working-directory: ./bot
-#      run: |
-#        pytest
\ No newline at end of file
+    - name: 'Create bot env file'
```
Comment (AkininD) [inline .github/workflows/python-package.yml:60]: Good!

**Review by AkininD — APPROVED** [review]

**Review by vadim328 — APPROVED** [review]
Everything is fine. Good implementation.

**Pull request merged**